client/web/src/views/ChartViewContent/ChartViewContent.tsx

```diff
@@ -33,31 +29,28 @@ export interface ChartViewContentProps {
  * Display chart content with different type of charts (line, bar, pie)
  */
 export const ChartViewContent: FunctionComponent<ChartViewContentProps> = props => {
-    const { content, className = '', history, viewID, telemetryService } = props
+    const { content, className = '', viewID, telemetryService } = props
 
-    const handleDatumLinkClick = useMemo(() => {
-        const nativeLinkHandler = createLinkClickHandler(history)
-
-        return (event: React.MouseEvent) => {
-            telemetryService.log('InsightDataPointClick', { insightType: getInsightTypeByViewId(viewID) })
-            nativeLinkHandler(event)
-        }
-    }, [viewID, telemetryService, history])
+    const handleDatumLinkClick = useCallback((): void => {
+        telemetryService.log('InsightDataPointClick', { insightType: getInsightTypeByViewId(viewID) })
+    }, [viewID, telemetryService])
 
     // Click link-zone handler for line chart only. Catch click around point and redirect user by
     // link which we've got from nearest datum point to user cursor position. This allows user
     // not to aim on small point on the chart and just click somewhere around the point.
-    const linkHandler = useMemo(() => {
-        const linkHandler = createProgrammaticLinkHandler(history)
-        return (event: DatumZoneClickEvent): void => {
+    const linkHandler = useCallback(
+        (event: DatumZoneClickEvent): void => {
             if (!event.link) {
                 return
             }
 
             telemetryService.log('InsightDataPointClick', { insightType: getInsightTypeByViewId(viewID) })
-            linkHandler(event.originEvent, event.link)
-        }
-    }, [history, viewID, telemetryService])
+
+            const url = new URL(event.link)
+            window.open(url.origin + url.pathname + url.search + url.hash, '_blank')?.focus()
```

Comment:
Is there any benefit in parsing the URL?
May also cause issues here because the link may be relative (?) and `new URL()` throws on relative URLs without supplying a base.

Also make sure to disallow `window.opener`: https://love2dev.com/blog/rel-noopener/
```suggestion
            window.open(event.link, '_blank', 'noopener')?.focus()
```